crates/cheatnet/src/forking/cache.rs

```diff
@@ -61,6 +64,10 @@ impl ForkCacheContent {
         // compiled_class_hash
         self.compiled_class_hash
             .extend(other.compiled_class_hash.clone());
+        // block_info
```

Comment:
```suggestion
```
Can you remove comments above as well? Those are kinda useless and only introduce unecessary mess.